index.js

```diff
@@ -42,8 +50,8 @@ module.exports = function transformJSONToTable(docs, options = {}) {
   // Go through each object again, this time, attempt to grab the value
   // At each possible path.
   let tableData = [allHeaders];
-  tableData     = tableData.concat(_.map(docs, doc => {
-    return _.map(allHeaders, header => {
+  tableData     = tableData.concat(_4_17_2.map(docs, doc => {
```

Comment:
I see we're using a native js concat here.  Would be better if we could utilize the _4_17_10.concat() in this case

Reply:
Nice catch. I'll address this when I can.